distribution/zip/ballerina-tools/resources/language-server-launcher.sh

```diff
@@ -0,0 +1,186 @@
+			 JAVA_HOME=$(/usr/libexec/java_home)
+           else
+             echo "Using Java version: $JAVA_VERSION"
+			 JAVA_HOME=$(/usr/libexec/java_home -v $JAVA_VERSION)
+		   fi
+	    fi
+        ;;
+esac
+
+# resolve links - $0 may be a softlink
+PRG="$0"
+
+while [ -h "$PRG" ]; do
+  ls=`ls -ld "$PRG"`
+  link=`expr "$ls" : '.*-> \(.*\)$'`
+  if expr "$link" : '.*/.*' > /dev/null; then
+    PRG="$link"
+  else
+    PRG=`dirname "$PRG"`/"$link"
+  fi
+done
+
+# Get standard environment variables
+PRGDIR=`dirname "$PRG"`
+
+# set BALLERINA_HOME
+BALLERINA_HOME=`cd "$PRGDIR/../../.." ; pwd`
+
+# For Cygwin, ensure paths are in UNIX format before anything is touched
+if $cygwin; then
+  [ -n "$JAVA_HOME" ] && JAVA_HOME=`cygpath --unix "$JAVA_HOME"`
+  [ -n "$BALLERINA_HOME" ] && BALLERINA_HOME=`cygpath --unix "$BALLERINA_HOME"`
+fi
+
+# For OS400
+if $os400; then
+  # Set job priority to standard for interactive (interactive - 6) by using
+  # the interactive priority - 6, the helper threads that respond to requests
+  # will be running at the same priority as interactive jobs.
+  COMMAND='chgjob job('$JOBNAME') runpty(6)'
+  system $COMMAND
+
+  # Enable multi threading
+  QIBM_MULTI_THREADED=Y
+  export QIBM_MULTI_THREADED
+fi
+
+# For Migwn, ensure paths are in UNIX format before anything is touched
+if $mingw ; then
+  [ -n "$BALLERINA_HOME" ] &&
+    BALLERINA_HOME="`(cd "$BALLERINA_HOME"; pwd)`"
+  [ -n "$JAVA_HOME" ] &&
+    JAVA_HOME="`(cd "$JAVA_HOME"; pwd)`"
+  # TODO classpath?
+fi
+
+if [ -z "$JAVA_HOME" ]; then
+  echo "JAVA_HOME is not set. Trying to find jre in BALLERINA_HOME.."
+  if [ -x "$BALLERINA_HOME/bre/lib/jre1.8.0_172/bin/java" ] ; then
+      JAVA_HOME="$BALLERINA_HOME/bre/lib/jre1.8.0_172"
+  else
+      echo "Failed find jre from BALLERINA_HOME."
+  fi
+fi
+
+if [ -z "$JAVACMD" ] ; then
+  if [ -n "$JAVA_HOME"  ] ; then
+    if [ -x "$JAVA_HOME/jre/sh/java" ] ; then
+      # IBM's JDK on AIX uses strange locations for the executables
+      JAVACMD="$JAVA_HOME/jre/sh/java"
+    else
+      JAVACMD="$JAVA_HOME/bin/java"
+    fi
+  else
+    JAVACMD=java
+  fi
+fi
+
+if [ ! -x "$JAVACMD" ] ; then
+  echo "Error: JAVA_HOME is not defined correctly."
+  exit 1
+fi
+
+
+
+# if JAVA_HOME is not set we're not happy
+if [ -z "$JAVA_HOME" ]; then
+  echo "You must set the JAVA_HOME variable before running Ballerina."
+  exit 1
+fi
+
+if [ $DEBUG_MODE = true ]; then
+  JAVA_DEBUG="-Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=$PORT"
+  echo "Please start the remote debugging client to continue..."
+else
+  JAVA_DEBUG=""
+fi
+
+JDK_18=`$JAVA_HOME/bin/java -version 2>&1 | grep -e "1.8."`
+if [ "$JDK_18" = "" ]; then
+    echo "Error: Ballerina is supported only on JDK 1.8"
+    exit 1
+fi
+
+for j in "$BALLERINA_HOME"/bre/lib/*.jar
+do
+    CLASSPATHS="$CLASSPATHS":$j
+done
+
+for j in "$BALLERINA_HOME"/lib/resources/composer/services/*.jar
+do
+    CLASSPATHS="$CLASSPATHS":$j
+done
```

Comment:
can change to
CLASSPATHS="$CLASSPATHS":$BALLERINA_HOME"/lib/resources/composer/services/*"

Reply:
Changed